labtool2.0/src/components/StudentTable.js

```diff
@@ -122,6 +122,14 @@ export const StudentTable = props => {
     }
   }
 
+  const changeFilterTag = () => {
+    return (e, data) => {
+      const { value } = data
+      const tag = props.tags.tags.find(tag => tag.id === value)
+      props.filterByTag(tag)
+    }
+  }
```

Comment:
This could be just
```
const changeFilterTag = (e, data) => {
      const { value } = data
      const tag = props.tags.tags.find(tag => tag.id === value)
      props.filterByTag(tag)
    }
```